### PR TITLE
made changes to allow all templates show BETA banner using super keyword

### PR DESCRIPTION
**Pull request by vaakinmade**

### CODE REVIEW

**Comment by AndyPorter** [comment]
this is dm front end, but lr_base_layout.html contains email address for digital register

File: application/templates/deednotfound.html [hunk 1]
```diff
@@ -2,6 +2,9 @@
 
 {% block content %}
 <div class="main" id="content">
+
+{{ super() }}
```
Comment (endamccormack) [inline application/templates/deednotfound.html]: 1. What is super for?
2. Why are we calling super in some places and not in others?
3. This super should be moved forward 1 indentation

Comment (vaakinmade) [inline application/templates/deednotfound.html]: so all the borrower-frontend template files extend the "base/lr_base_layout.html”, the super keyword allows the block_content to inherit from its parent block in the "base/lr_base_layout.html” which contains the beta banner elements. All template files  use the super function, nice clean way to include the beta banner across the board.

**Comment by endamccormack** [comment]
:+1:

**Pull request merged**